colors/simple.kak

```diff
@@ -0,0 +1,46 @@
+# Kakoune default color default
+
+# For default
+face value default
+face type default
+face identifier default
+face string blue
+face keyword default
+face operator default
+face attribute default
+face comment blue
+face meta default
+face builtin default
+
+# For default
+face title default
+face header default
+face bold default
+face italic default
+face mono default
+face block default
+face link blue
+face bullet default
+face list default
+
+# builtin default
+face Default default,default
+face PrimarySelection white,blue
+face SecondarySelection black,blue
+face PrimaryCursor black,white
+face SecondaryCursor black,white
```

Comment:
Where is `{Primary,Secondary}CursorEol`? https://github.com/mawww/kakoune/blob/master/colors/default.kak#L34

Also: you have the same colour for `PrimaryCursor` and `SecondaryCursor`. Then with multiple selections of length the user won't be able to discern which is the primary cursor.

Reply:
> Also: you have the same colour for PrimaryCursor and SecondaryCursor. Then with multiple selections of length the user won't be able to discern which is the primary cursor.

On a side note, do you really need to know which cursor is the primary one?

Reply:
Yes, to use some features reliably you do, some that come to mind are completions in insert mode, `vc`, `*`, `N`, `<space>`, `(`, well look for main in https://github.com/mawww/kakoune/blob/master/doc/pages/keys.asciidoc

Reply:
Heh, never noticed anything like that, but you're right

Reply:
[done](https://github.com/mawww/kakoune/pull/3124/commits/e1b81d4ac6e28b43045e8778e18c94b4db11dc29)